### PR TITLE
Update file BBEs with new API changes

### PR DESCRIPTION
**Pull request by ThishaniLucas**
$subject.

### CODE REVIEW

File: examples/file/file.bal [hunk 1]
```diff
@@ -1,74 +1,77 @@
...
     }
 
     // Get metadata information of the file.
-    file:FileInfo|error fileInfoResults = file:getFileInfo("bar.txt");
-    if (fileInfoResults is file:FileInfo) {
-        io:println("File name: ", fileInfoResults.getName());
-        io:println("File size: ", fileInfoResults.getSize().toString());
-        io:println("Is directory: ", fileInfoResults.isDir().toString());
-        io:println("Modified at ", fileInfoResults.getLastModifiedTime().
+    file:MetaData|error fileInfoResults = file:getMetaData("bar.txt");
```
Comment (daneshk) [inline examples/file/file.bal:28]:
```suggestion
    file:MetaData|error fileMetadata = file:getMetaData("bar.txt");
```

**Review by daneshk — APPROVED** [review]

**Comment by niveathika** [comment]
@ThishaniLucas Seems the build is failing, could you take a look?

**Comment by ThishaniLucas** [comment]
@niveathika I already noticed the failure. It's coming from the task :ballerina:testDevTools. The same failure is there in other PRs as well. Skipping this task will make the build pass locally.

**Comment by niveathika** [comment]
This task is introduced by https://github.com/ballerina-platform/ballerina-distribution/pull/922 by Test framework team. Let's coordinate with them and disable this temporarily if possible.

**Pull request merged**